contracts/test/TestPaymasterWithPostOp.sol

```diff
@@ -20,7 +21,8 @@ contract TestPaymasterWithPostOp is TestPaymasterAcceptAll {
         return ("1", SIG_VALIDATION_SUCCESS);
     }
 
-    function _postOp(PostOpMode, bytes calldata, uint256, uint256)
+    function _postOp(PostOpMode, bytes calldata, uint256 actualGasCost, uint256)
     internal override {
+        emit PostOpActualGasCost(actualGasCost);
```

Comment:
This new emit breaks the gas-calc, as it adds another >1000 gas to the postOp.
The gas-calc for paymaster+postOp raised by 1200, while the cost of the EntryPoint code change (for postOp penalty cost) was about 150

We should have a separate GasCalcPaymasterWithPostOp.

Reply:
Actually I removed its usage so it can be removed

Reply:
that would be nice, though I currently see it is referenced from a test.

Reply:
Added a gas calc paymaster